org.flatpak.popout3d.yml

```diff
@@ -0,0 +1,204 @@
+app-id: com.github.popout3-apps.popout3d
```

Comment:
I have seen hyphens in in names like org.kde.WaylandDecoration.QGnomePlatform-decoration, so I assumed it was alright. Is it just the first name, the name of the Git domain that can't have a hyphen?
If so, I'll change it and everything else

Reply:
It is allowed in the last component of the id. :-/

Reply:
I'm changing it to PopoutApps as '_' isn't allowed in the GitHub domain/account name.

Reply:
For the record, I never said to change the GitHub account. Just to replace `-` with `_` in the app id.

Reply:
I thought it had to be the same, and even if not, I thought it would be best not to cause confusion by having different names. Hopefully changing it doesn't cause any problems.